app/views/katello/api/v2/activation_keys/base.json.rabl

```diff
@@ -7,6 +7,30 @@ node :multi_content_view_environment do |ak|
   ak.multi_content_view_environment?
 end
 
+child :content_view_environments => :content_view_environments do
+  node :content_view do |cve|
+    {
+      id: cve.content_view&.id,
+      name: cve.content_view&.name,
+      composite: cve.content_view&.composite,
+      content_view_version: cve.content_view_version&.version,
+      content_view_version_id: cve.content_view_version&.id,
+      content_view_version_latest: cve.content_view_version&.latest?,
+      content_view_default: cve.content_view&.default?
+    }
+  end
+  node :lifecycle_environment do |cve|
+    {
+      id: cve.lifecycle_environment&.id,
+      name: cve.lifecycle_environment&.name,
+      lifecycle_environment_library: cve.lifecycle_environment&.library?
+    }
+  end
+  node :candlepin_name do |cve|
+    cve.candlepin_name
```

Comment:
It's also called just `label`..

I've always been fond of "candlepin_name" :cry:  but I guess I can change it.

Reply:
Time to make `candlepin_name` `katello_name` since we are using it everywhere  including asking users to use it in hammer..
I'll leave it to your better judgement..I know we have a story to make these candlepin_names more easily available to users..